spec/menu_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'menu'
+
+describe Menu do
+  it 'has a list of items by default' do
```

Comment:
Down the line, would there potentially be scope to change this test so it raises an error if `@food` is empty?